app/src/components/cards/itemCard.tsx

```diff
@@ -9,9 +9,9 @@ import {
 import styles from "./Card.module.css";
 
 const ItemCard = ({ id, isLargerThanLargeTablet, item }) => {
-  return (
+  const truncatedTitle = item.title.length > 80; // Pretty much random
```

Comment:
Declare `80` as a constant variables to not have magic numbers, even if it's random.